README.md

```diff
@@ -1,19 +1,14 @@
 # ![nf-core/rnaseq](docs/images/nf-core-rnaseq_logo_light.png#gh-light-mode-only) ![nf-core/rnaseq](docs/images/nf-core-rnaseq_logo_dark.png#gh-dark-mode-only)
 
-[![GitHub Actions CI Status](https://github.com/nf-core/rnaseq/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/rnaseq/actions?query=workflow%3A%22nf-core+CI%22)
-[![GitHub Actions Linting Status](https://github.com/nf-core/rnaseq/workflows/nf-core%20linting/badge.svg)](https://github.com/nf-core/rnaseq/actions?query=workflow%3A%22nf-core+linting%22)
```

Comment:
what happend to the GHA badges?

Reply:
Looks like they aren't in the template by default anymore and need to be added explicitly. I have deleted the template update branch for this PR but you can see the same here:
https://github.com/nf-core/viralrecon/tree/nf-core-template-merge-2.5.1